include/bzfsAPI.h

```diff
@@ -275,6 +275,10 @@ typedef enum
   bz_eGameResumeEvent,
   bz_eGameStartEvent,
   bz_eGameEndEvent,
+  bz_ePollStartEvent,
+  bz_ePollVoteEvent,
+  bz_ePollVetoEvent,
+  bz_ePollEndEvent,
```

Comment:
Doesn't this change require incrementing the API version? Previously compiled plugins are likely to be incompatible.